package.json

```diff
@@ -55,7 +55,7 @@
     "ejson": "2.2.0",
     "eslint-config-airbnb": "^18.1.0",
     "expo-apple-authentication": "^2.2.1",
-    "expo-av": "8.2.1",
+    "expo-av": "^8.2.1",
```

Comment:
Unnecessary.

Reply:
I upgraded it because 8.2.1 had some bugs with iOS background playback https://github.com/expo/expo/issues/5472#issuecomment-663606417